x-pack/plugins/enterprise_search/public/applications/workplace_search/views/content_sources/components/overview.tsx

```diff
@@ -268,9 +241,6 @@ export const Overview: React.FC = () => {
                       <strong>{group.name}</strong>
                     </EuiText>
                   </EuiFlexItem>
```

Comment:
I wonder if we could also remove EuiFlexGroup and EuiFlexItem that are wrapping group name?